its/sources/sonar-project.properties

```diff
@@ -0,0 +1,5 @@
+sonar.projectKey=xml-ruling
```

Comment:
why this file? you are setting everything in the test

Reply:
For manual testing on local instance, no need to configure anything. just run sonar scanner from there.